pyrdp/player/ReplayTab.py

```diff
@@ -51,6 +61,20 @@ def __init__(self, fileName: str, parent: QWidget):
 
         self.tabLayout.insertWidget(0, self.controlBar)
 
+    def makeThumbnailsMap(self, thumbnail_directory):
+        thumbnails = {}
+        first_thumbnail_timestamp = 0
+        if thumbnail_directory:
+            for i, file in enumerate(os.listdir(thumbnail_directory)):
+                if file.endswith(".png"):
+                    timestamp = int(file.replace('.png', ''))
+                    if i == 0:
+                        first_thumbnail_timestamp = timestamp
+                    thumbnails[timestamp - first_thumbnail_timestamp] = (
+                    f"{thumbnail_directory}/{file}",
+                    f"{thumbnail_directory}/gdi_cache/{file}".replace('.png', '.bitmapcache'))
```

Comment:
I don't think saving a snapshot of the cache at every frame is going to be enough. on large enough intervals, the cache might change several times in-between two thumbnails.

Reply:
Yeah, but if I save the cache at the time of the thumbnail, and i replay events starting at the thumbnail, shoulnt the cache get updated correctly then? Or the player doesnt update the cache when playing the replay?